mcs/class/corlib/System.Security.Policy/Publisher.cs

```diff
@@ -38,7 +38,8 @@ namespace System.Security.Policy {
 	public sealed class Publisher :
 		EvidenceBase,
 		IIdentityPermissionFactory, IBuiltInEvidence {
-	
+
+		[NonSerialized]
```

Comment:
Why did you change file which is no longer used?

Reply:
Left-over from testing, before replacing the file.

Reply:
Could you delete the no longer used files